tests/parse/test_parse_case.py

```diff
@@ -1,17 +1,12 @@
-import pytest
-
 from datetime import datetime
 from pprint import pprint as pp
-from scout.parse.case import (
-    parse_case,
-    parse_ped,
-    parse_individuals,
-    parse_individual,
-    parse_case_data,
-    removeNoneValues,
-)
-from scout.exceptions import PedigreeError
+
+import pytest
+
 from scout.constants import REV_SEX_MAP
+from scout.exceptions import PedigreeError
+from scout.parse.case import (parse_case, parse_case_data, parse_individual,
```

Comment:
It’s interesting to watch the different formatting styles evolve from commit to commit. 😆